clients/web/src/templates/widgets/editAssetstoreWidget.jade

```diff
@@ -14,6 +14,9 @@
             .form-group
               label.control-label(for="g-edit-fs-root") Root directory
               input.input-sm#g-edit-fs-root.form-control(type="text", placeholder="Root directory")
+            .form-group
+              label.control-label(for="g-edit-fs-perms") Default file permissions
+              input.input-sm#g-edit-fs-perms.form-control(type="text", placeholder="Default permissions")
```

Comment:
Let's make the placeholder be the actual default value (600), and make the label say "File creation permissions (octal)"